### DISCUSSION TITLE
add new exchanges - interest, procedure, integration test

### DISCUSSION THREAD

**Discussion by pte1601** [bmoscon/cryptofeed#875] [Ideas]
Dear cryptofeed users
thanks for this concise and amazing project.
I am interested to contribute adding certain exchange feeds.

1. Do you have certain guidelines what features the exchanges should support?
2. Should I post some exchange names and then let's agree on them? 

Thanks for helping out here

Greetings, Peter

**Comment by bmoscon** [comment]
hi @pte1601 - if you look at any of the exchanges or any of the parent classes you'll see what they support. Let me know if you have any questions.
